app/browser/api/ledger.js

```diff
@@ -2366,15 +2391,14 @@ const initialize = (state, paymentsEnabled) => {
         }
         return
       }
-
       fs.readFile(pathName(statePath), (err, data) => {
         if (err) {
           module.exports.disablePayments()
           return console.error('read error: ' + err.toString())
         }
 
         try {
```

Comment:
Can we move this whole try/catch into `onInitReadAction`

Reply:
++